### PR TITLE
HDDS-1919. Fix Javadoc in TestAuditParser

### PR DESCRIPTION
**Pull request by pingsutw**

### CODE REVIEW

File: hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java [hunk 1]
```diff
@@ -42,7 +42,7 @@
 import java.util.List;
 
 /**
- * Tests GenerateOzoneRequiredConfigurations.
+ * Tests TestAuditParser.
```
Comment (bharatviswa504) [inline hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java]: It should be AuditParser.

Comment (pingsutw) [reply hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java]: @bharatviswa504 Thanks for your review 
I already updated my patch

File: hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java [hunk 2]
```diff
@@ -42,9 +42,9 @@
...
  */
-public class TestAuditParser {
+public class AuditParser {
```
Comment (bharatviswa504) [inline hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java]: Hi @pingsutw 
The class name should be TestAuditParser.
Comment for the class should be Test AuditParser.

Comment (pingsutw) [reply hadoop-ozone/tools/src/test/java/org/apache/hadoop/ozone/audit/parser/TestAuditParser.java]: Sorry for my mistake, I have updated patch

**Comment by bharatviswa504** [comment]
Merging this without CI, as it updates only Javadoc comment.

**Comment by pingsutw** [comment]
@bharatviswa504 Thank you so much

**Pull request merged**